src/mazela-artur/Fsd.Artur.Cs2/Fsd.Artur.Cs2/ConsoleActions/Readers/AnimalReader.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fsd.Artur.Cs2.ConsoleActions.Readers
+{
+    public class AnimalReader : IAnimalReader
```

Comment:
Wszystko bardzo dobrze tutaj. Taki lewej wyżej to nazwać tą klasę ConsoleAnimalReader. Dzięki temu, że implementuje interfejs IAnimalReader, w pozostałych częściach aplikacji używamy typu interfejsu a nie klasy. Gdy nagle zmienimy tą aplikację na web, wystarczy podmienić ConsoleAnimalReader na WebAnimalReader i nie trzeba modyfikować żadnego innego miejsca. Będę jeszcze o tym mówił